### ISSUE TITLE
Unable to see connection tree

### ISSUE THREAD

**Issue by MatCast** [mtxr/vscode-sqltools#878]
**Describe the bug**
When clicking on the left arrow on an active connection i get the following error and the connection tree is not shown:
`Request connection/GetChildrenForTreeItemRequest failed unexpectedly without providing any details.`

After the error appeared once it does not show up again upon re-clicking on the arrow but the connection tree still does not diplay.

I can run queries on the DB normally.

**Expected behavior**
The connection tree is shown when clicking on the connection arrow.

**Screenshots**
![image](https://user-images.githubusercontent.com/32680270/141263000-ae8b5e6b-135d-4e8c-8298-f1c771c8f8fc.png)

**Desktop (please complete the following information):**
 - SQLTools Version v0.23.0
 - VSCode Version: 1.62.1
 - OS: Windows 10
 - Driver: 
   - [x] MSSQL/Azure
 - Database version: Microsoft Azure SQL Data Warehouse - 10.0.13655.0

**Comment by tarekahf** [comment]
I have the same problem. Any resolution found?

**Comment by tkekawa3712** [comment]
Any resolution?

**Comment by gjsjohnmurray** [comment]
After connecting are you able to run this query?
```sql
SELECT * FROM INFORMATION_SCHEMA.TABLES
```
If so, does this list schemas and tables?

**Comment by git1453** [comment]
I have the same problem.
And when I want to run this query

> SELECT * FROM INFORMATION_SCHEMA.TABLES

Then 
<img width="401" alt="image" src="https://user-images.githubusercontent.com/83624279/193221667-ade50c62-ba84-4789-8f7d-a3267346f99a.png">

**Comment by gjsjohnmurray** [comment]
@git1453 please confirm which driver you are using, and which versions of the driver and main extension.

Does the CONSOLE button give any more information?

And if you right-click on your connection in the SQLTools view, choose "Edit Connection", then press "TEST CONNECTION" does this succeed?

**Comment by packyzbq** [comment]
@gjsjohnmurray 
I have the same problem.
And when I run the query
> SELECT * FROM INFORMATION_SCHEMA.TABLES

Then I get this
![image](https://user-images.githubusercontent.com/6461656/201592407-b9929faf-f633-4701-9a7f-e485e7c9b7d4.png)

The console output is 
> Connection is closed.

By the way,  
SQLTools Version v0.26.0
SQLTools Microsoft SQL Server/Azure: v0.3.0
VSCode Version: 1.73.1
OS: Windows 10
Driver:
 MSSQL/Azure
Database version: Microsoft SQLServer 2014

**Comment by gjsjohnmurray** [comment]
@packyzbq I don't have access to an instance of SQL Server 2014 to test against.

If you take SQLTools out of the equation by opening a native MSSQL command shell on that server and database, using the same userid as your SQLTools connection specifies, does that statement work?

**Comment by packyzbq** [comment]
> @packyzbq I don't have access to an instance of SQL Server 2014 to test against.
> 
> If you take SQLTools out of the equation by opening a native MSSQL command shell on that server and database, using the same userid as your SQLTools connection specifies, does that statement work?

@gjsjohnmurray 
YES.  And I also tried running this statement on both DataGrip and SQL Server extension for Visual Studio Code with the same userid, and it worked fine.

**Comment by pallapo** [comment]
try to increase connection timeout, for me 50000 worked fine.

**Comment by tarekahf** [comment]
> try to increase connection timeout, for me 50000 worked fine.

How to increase the timeout?

**Comment by SecondSite-web** [comment]
I had the same problem now. Solved it by unchecking the 'useUTC' option in VSCode - SQL Tools Settings in the connection settings

**Comment by jasonedokpa** [comment]
I tried the same, and sadly, that did not work for me.

**Comment by shalevku** [comment]
Same problem here, connection tree doesn't expand but queries work including the query that [gjsjohnmurray](https://github.com/gjsjohnmurray) suggested to try:
> SELECT * FROM INFORMATION_SCHEMA.TABLES

**Comment by csubhodeep** [comment]
Hello all, do we have any workaround for this one?

**Comment by fzhem** [comment]
Related to #710

**Comment by fzhem** [comment]
@csubhodeep  The sisue occurs due to how list of databases is retrieved. I've fixed this issue in my fork:

https://marketplace.visualstudio.com/items?itemName=fzhem.sqltools-driver-mssql-commver

**Comment by Lument-AndrewExley** [comment]
Can concur same experience with "Request connection/GetChildrenForTreeItemRequest failed unexpectedly without providing any details."

@fzhem Thanks for identifying and offering a fork, unfortunately, being on Apple Silicon, it isn't supported for me.

**Comment by fzhem** [comment]
> Can concur same experience with "Request connection/GetChildrenForTreeItemRequest failed unexpectedly without providing any details."
> 
> @fzhem Thanks for identifying and offering a fork, unfortunately, being on Apple Silicon, it isn't supported for me.

@Lument-AndrewExley  The fork currently supports the msnodesqlv8 driver, which has limited platform compatibility. Given sufficient interest, I may make it universal by disabling msnodesqlv8 on unsupported platforms. This will take time alongside other projects and day-to-day work, but I can create a pull request to address the issue in the official driver in the meantime.

**Comment by fzhem** [comment]
@Lument-AndrewExley It's fixed in the latest version, can you give it a go?
